src/H3MP/Plugin.cs

```diff
@@ -334,7 +346,83 @@ private void Start()
 				Logger.LogDebug("Autostarting host from game launch...");
 
 				StartCoroutine(_Host());
+			}					
+		}
+
+		private void OnSceneLoaded(Scene scene, LoadSceneMode mode)
+		{
+			if (scene.name == "MainMenu3")
+			{
+				// Creates a cloned mainscreen panel for H3MP changelog
+				CreatePanel();
+
+				// Gets body text from GitHub to populate H3MP panel
+				StartCoroutine(SendGetCoroutine());
 			}
+
+			IEnumerator SendGetCoroutine()
+			{
+				var needsUpdate = false;
+				var currentVersion = "v" + _version.ToString();
+				var latestVersion = currentVersion;
+
+				// Get the latest release from GitHub
+				using (UnityWebRequest www = UnityWebRequest.Get("https://api.github.com/repos/ash-hat/h3mp/releases/latest"))
```

Comment:
`ash-hat/h3mp` gets used twice, might be a good idea to make it a constant in case repository path changes.